src/server/redis_connection.h

```diff
@@ -74,7 +74,12 @@ class Connection : public EvbufCallbackBase<Connection> {
   std::string SizeOfSet(T len) const {
     return protocol_version_ == RESP::v3 ? "~" + std::to_string(len) + CRLF : MultiLen(len);
   }
-  std::string ArrayOfSet(const std::vector<std::string> &elems) const;
+  std::string SetOfBulkStrings(const std::vector<std::string> &elems) const;
+  template <typename T, std::enable_if_t<std::is_integral_v<T>, int> = 0>
+  std::string SizeOfMap(T len) const {
```

Comment:
From my perspective, the function name can be something like `StartMap` or `MapHeader`.

Reply:
How about changing them to `StartOutputOfMap` or `StartMapOutput` due to  `conn->StartMap` feels like we will create a map inside the connection at first look.

Reply:
I don't think `Output` is a better idea. It seems like an IO operation, rather than a no-side-effect string operation.

If `StartMap` looks weird to you, maybe we can try `MapHeader`?